Content.Shared/_RMC14/Xenonids/Egg/XenoEggSystem.cs

```diff
@@ -519,4 +526,44 @@ public override void Update(float frameTime)
             SetEggState((uid, egg), XenoEggState.Grown);
         }
     }
+
+    private void OnGetVerbs(Entity<XenoEggComponent> ent, ref GetVerbsEvent<ActivationVerb> args)
+    {
+        var uid = args.User;
+
+        // if it doesn't have an actor and we can't reach it then don't add the verb
+        if (!EntityManager.TryGetComponent(uid, out ActorComponent? actor))
+            return;
+
+        if (!EntityManager.TryGetComponent(uid, out GhostComponent? ghostComp))
```

Comment:
Entity systems have a short-hand format for component methods
```suggestion
        if (!TryComp(uid, out ActorComponent? actor))
            return;

        if (!TryComp(uid, out GhostComponent? ghostComp))
```